.github/workflows/ci-workflow.yml

```diff
@@ -12,37 +12,6 @@ on:
       - develop
 
 jobs:
-  nanos_build:
-    name: Build app for NanoS
-    runs-on: ubuntu-latest
-    container:
-      image: ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder:latest
-    steps:
-      - name: Clone
-        uses: actions/checkout@v2
-      - name: Build
-        run: make
-      - name: Upload app binary
-        uses: actions/upload-artifact@v2
-        with:
-          name: celo-app
-          path: bin
-
-  nanox_build:
-    name: Build app for NanoX
-    runs-on: ubuntu-latest
-    container:
-      image: ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder:latest
-    steps:
-      - name: Clone
-        uses: actions/checkout@v2
-      - name: Build
-        run: make BOLOS_SDK=$NANOX_SDK
-      - name: Upload app binary
-        uses: actions/upload-artifact@v2
-        with:
-          name: celo-app
-          path: bin
 
   job_scan_build:
```

Comment:
Can you rather use the guideline enforcer ? https://github.com/LedgerHQ/app-boilerplate/blob/master/.github/workflows/guidelines_enforcer.yml
It part of it?